README.md

```diff
@@ -83,7 +85,7 @@ The default pipeline loaded with this Jenkins instance will automatically run th
 
 1. Retrieve latest code of the promotion app from Git​
 2. Build artifact package (latest build)​
-3. Upload Build to S3 repository​
+3. Upload build to S3 repository​
```

Comment:
"the build" or the "artifact package"?

Reply:
Dunno - I'm new here, just going through the tutorial. Please take it with someone who knows what he's talking about :)

Reply:
Several minor changes:

Git should be GitHub
Add a period to each step.
change build to artifact package.

Reply:
> "the build" or the "artifact package"?

Artifact package

Reply:
> Git should be GitHub
Actually, no. In this case sources are taken from git repo created on Azure DevOps. So, I think we can just use "Git repo" instead of "Git"

Reply:
ok. Thanks for clarifying Dmytro.

Reply:
So... we're good? If not, please fix this, as I must move on to my "real" job and leave the writing to the writing experts :)